src/index.ts

```diff
@@ -105,6 +106,7 @@ export const supplyFetchers: Record<string, SupplyFetcher> = {
   "577f0b1342f8f8f4aed3388b80a8535812950c7a892495c0ecdf0f1e0014df10464c4454":
     fldtFetcher,
   f43a62fdc3965df486de8a0d32fe800963589c41b38946602a0dc53541474958: agixFetcher,
+  "420000029ad9527271b1b1e3c27ee065c18df70a4a4cfc3093a41a44": axoFetcher,
```

Comment:
`420000029ad9527271b1b1e3c27ee065c18df70a4a4cfc3093a41a4441584f`

Reply:
thank you